node_modules/oae-activity/tests/test-email.js

```diff
@@ -1384,10 +1321,11 @@ describe('Activity Email', function() {
             var secondUser = users[userIds[1]];
             var thirdUser = users[userIds[2]];
 
-            // Clear the email for the third user so we have a user that has no email address
-            RestAPI.User.updateUser(thirdUser.restContext, thirdUser.user.id, {'email': 'notavalidemail'}, function(err, updatedThirdUser) {
+            // Clear the email for the third user so we have a user that has no email address. This situation
+            // can arise when somebody logs in via Shibboleth, the email attribute isn't released and the user
+            // doesn't fill it in
```

Comment:
Ideally, no. But with all the authentication strategies we have going on, it might be better to be prudent and make sure things don't break if a user does manage to slip in